package.json

```diff
@@ -53,6 +53,7 @@
     "@ngrx/store": "^15.2.0",
     "@ngrx/store-devtools": "^15.2.0",
     "@ngx-translate/core": "^14.0.0",
+    "buffer": "^6.0.3",
```

Comment:
I think there is already some “Buffer” functionality which comes from node so do we need to install that extra dependency? Can’t be reused that version of Buffer from node?